### ISSUE TITLE
bug:  S3 - public access block resulting failed

### ISSUE THREAD

**Issue by coobachey** [aquasecurity/defsec#890]
**Describe the bug**
in 1.27.1 - s3 check for ACl passed s3 blocks in test
now 1.27.2  ++. defsec  v0.71.11 v0.72.0. Alerts to the issues:

5 results are posted 
ID aws-s3-specify-public-access-block
ID aws-s3-no-public-buckets
ID aws-s3-ignore-public-acls
ID aws-s3-block-public-policy
ID aws-s3-block-public-acls

**To Reproduce**

test the TF code below - with or without count statement

**Expected behavior**

**Output of your tfsec command with --debug flag**

**System Info**

 - tfsec version: `1.27.2`
 - terraform version: `any above 1.2.x`
 - OS: `NA`
**Example Code**

```
resource "aws_s3_bucket" "s3_bucket" {
  bucket = var.s3_bucket_name
  tags = var.tags
}


resource "aws_s3_bucket_public_access_block" "s3_bucket" {
  count = var.dev-deploy ? 1 : 0
  bucket = aws_s3_bucket.s3_bucket.id
  block_public_acls   = true
  block_public_policy = true
  ignore_public_acls  = true
  restrict_public_buckets = true
}
resource "aws_s3_bucket_acl" "s3_bucket_acl" {
  bucket = aws_s3_bucket.s3_bucket.id
  acl    = "private"
}
```

**Comment by coobachey** [comment]
if I hardcode Bucket name &<or> set a default value for var.  this passes. 

variable "s3_bucket_name" {
  description = "The name of the S3 bucket to create"
  default = "something"
}

**Comment by liamg** [comment]
This has now been fixed in v0.72.1 :+1:
